src/lib.rs

```diff
@@ -158,22 +160,19 @@ where
 /// the type inside the empty slice.
 pub const EMPTY_TAGS: &[&str] = &[];
 
-/// The Datadog type is the main entry point for the library
-pub type Datadog = DatadogWrapper<dogstatsd::Client>;
```

Comment:
Wasn't this made to be non-breaking? Why was it removed?

Reply:
I think it's non breaking in e.g. the benchmark (calling `Datadog::init` still works exactly like it did before). My understanding is that the `impl Datadog<dogstatsd::Client>` block (which just wraps a generic inner, and provides `init`, `incr`, etc) is what's being used when you call e.g. `Datadog::init` or any other `pub fn` we expose, and since they're only defined on the non-generic `impl Datadog<dogstatsd::Client>`, so it breaks nothing, but lets us use some other `Datadog<C>` in our tests, calling e.g. the private `do_X` methods directly

Reply:
Can confirm that it works as expected

Reply:
Ok great! Thanks for the ELI5 :heart: